### PR TITLE
rename symbol names to be consistent with tipb

### PR DESCRIPTION
**Pull request by dragonly**
since the generated code for `ReportCPUTime` will be `ReportCpuTime` for rust, we should leave the naming to the corresponding language protoc extension.

### CODE REVIEW

**Comment by zhongzc** [comment]
Have you run `make go`?

**Comment by dragonly** [comment]
> Have you run `make go`?

Sorry but no, fixed in the next commit.

**Review by zhongzc — APPROVED** [review]

File: proto/resource_usage_agent.proto [hunk 1]
```diff
@@ -16,13 +16,13 @@ option java_package = "org.tikv.kvproto";
...
 
-message ReportCpuTimeRequest {
+message CPUTimeRecord {
     bytes resource_group_tag = 1;
 
-    // List<(UnixTimestamp, CpuTime)>
+    // The following 2 repeated zipped together represents a List<(UnixTimestamp, CPUTime)>
 
     // UNIX timestamp in second.
     repeated uint64 record_list_timestamp_sec = 2;
```
Comment (breeswish) [inline proto/resource_usage_agent.proto:28]: Should we unify these field names as well?

**Review by breeswish — APPROVED** [review]

**Comment by breeswish** [comment]
Defer until 5.1 is released.

File: proto/resource_usage_agent.proto (hunk 1 above)
Comment (dragonly) [reply proto/resource_usage_agent.proto:28]: done in https://github.com/pingcap/tipb/pull/233

**Review by breeswish — APPROVED** [review]

**Review by crazycs520 — APPROVED** [review]

**Comment by breeswish** [comment]
merge with a TiDB PR.

**Pull request merged**